esxi/assets/configuration/spec.yaml

```diff
@@ -42,4 +42,22 @@ files:
           value:
             type: boolean
             example: false
+        - name: excluded_host_tags
+          description: |
+            The ESXi integration collects tags for every ESXi host or VM in your environment.
+            Those tags can be `esxi_url`, esxi_type`, `esxi_host`, `esxi_folder`, `esxi_cluster`
+            `esxi_compute`, `esxi_datacenter`, `esxi_datastore` and are attached to the ESXi metrics by default.
+            By default these tags are submitted as host tags, but you can submit them as metric tags by using
```

Comment:
This part `but you can submit them as metric tag` might be misleading, since using this parameter will move these host tags to metrics tags.